js/src/personal/maliha/Social.tsx

```diff
@@ -1,56 +1,86 @@
+/* Social Media Icons */
 import { Anchor, Stack } from '@mantine/core'
 import { CgMail } from 'react-icons/cg'
 import { CiLinkedin } from 'react-icons/ci'
-import { FaInstagram, FaGithub } from 'react-icons/fa'
+import { FaInstagram, FaGithubSquare } from 'react-icons/fa'
 import { useHover } from '@mantine/hooks'
 import styles from './Maliha.module.css'
 
 const iconMap = [
   {
     icon: FaInstagram,
-    size: '4.5em',
+    size: '3.5em',
     href: 'https://www.instagram.com/birb.erry/',
+    outlineColor: '#000000',
+    outlineWidth: '.5em',
   },
   {
-    icon: FaGithub,
-    size: '4.75em',
+    icon: FaGithubSquare,
+    size: '3.5em',
     href: 'https://github.com/MalihaT111',
-  },
-  {
-    icon: CiLinkedin,
-    size: '5em',
-    href: 'https://www.linkedin.com/in/maliha-tasnim-81864b178/',
+    outlineColor: '#000000',
+    outlineWidth: '.4em',
   },
   {
     icon: CgMail,
-    size: '5em',
+    size: '4em',
     href: 'mailto:ayesiddika@gmail.com',
+    outlineColor: '#000000',
+    outlineWidth: '0.01em',
+  },
+  {
+    icon: CiLinkedin,
+    size: '4em',
+    href: 'https://www.linkedin.com/in/maliha-tasnim-81864b178/',
+    outlineColor: '#000000',
+    outlineWidth: '0.01em',
   },
 ]
 
```

Comment:
It's good to add comments above components with the general idea of what they do.